src/CVSelect/ContentViewList.js

```diff
@@ -39,7 +39,19 @@ export default function ContentViewList() {
     window.location.assign(`/content-view/${id}`);
   };
 
-  const handleOnSearchChange = () => {};
+  const handleOnSearchChange = (val) => {
+    if (val === "") {
```

Comment:
Instead of checking if it's "", you should just check !val. This will also handle if val is undefined (which it shouldn't be, but it's better to handle it in case that ever changes).